libs/labelbox/src/labelbox/schema/project.py

```diff
@@ -1226,6 +1249,50 @@ def get_queue_mode(self) -> "QueueMode":
         else:
             raise ValueError("Status not known")
 
+    def add_model_config(self, model_config_id: str) -> str:
+        """ Adds a model config to this project.
+
+        Args:
+            model_config_id (str): ID of a model config to add to this project.
+
+        Returns:
+            str, ID of the project model config association. This is needed for updating and deleting associations.
+        """
+
+        query = """mutation CreateProjectModelConfigPyApi($projectId: ID!, $modelConfigId: ID!)  {
+                    createProjectModelConfig(input: {projectId: $projectId, modelConfigId: $modelConfigId}) {
+                        projectModelConfigId
+                    }
+                }"""
+
+        params = {
+            "projectId": self.uid,
+            "modelConfigId": model_config_id,
+        }
+        result = self.client.execute(query, params)
+        return result["createProjectModelConfig"]["projectModelConfigId"]
+
+    def remove_project_model_config(self, project_model_config_id: str) -> bool:
```

Comment:
why not `delete_...` as consistent with other method names?

Reply:
Fixed.